ipn/prefs_test.go

```diff
@@ -24,7 +24,7 @@ func fieldsOf(t reflect.Type) (fields []string) {
 func TestPrefsEqual(t *testing.T) {
```

Comment:
Can you also add a couple of test cases for Hostname comparison, similar to the other cases already in there?

Reply:
Done.